packages/shared/blockchain/evm-based/src/lib/transactions/custodial/index.ts

```diff
@@ -52,6 +152,51 @@ export const custodial = (args: {
         body: ChainGenerateCustodialAddress,
         provider?: string,
       ) => generateCustodialWallet(body, args.web3, provider),
```

Comment:
I would delete this generateWallet call - its old and deprecated and only *Batch one should be used